src/pages/settings/bank-accounts/edit/Edit.tsx

```diff
@@ -108,23 +114,20 @@ const Edit = () => {
       onCancelClick={handleCancel}
       onSaveClick={handleSave}
     >
-      <Card title={t('edit_bank_account')}>
-        <form
-          onSubmit={(e) => e.preventDefault()}
-          ref={bankAccountForm}
-          className="my-6 space-y-4"
-        >
-          <Element leftSide={t('bank_account_name')}>
-            <InputField
-              id="bank_account_name"
-              value={form?.values?.bank_account_name}
-              onChange={form?.handleChange}
-              errorMessage={errors?.bank_account_name}
-            />
-          </Element>
-        </form>
+      <Card onFormSubmit={handleSave} title={t('edit_bank_account')}>
+        <Element leftSide={t('bank_account_name')}>
+          <InputField
+            value={accountDetails?.bank_account_name}
+            onChange={(event: FormEvent<HTMLFormElement>) =>
+              handleChange(
+                'bank_account_name',
+                (event?.target as HTMLInputElement)?.value
+              )
+            }
```

Comment:
Unless absolutely needed, please don't use platform onChange event. InputField component exposes onValueChange so you can make use of that.

Reply:
> Unless absolutely needed, please don't use platform onChange event. InputField component exposes onValueChange so you can make use of that.

sure, I missed that, changed